### PR TITLE
VueJS example

### PR DESCRIPTION
**Pull request by andreahaku**
Adds VueJS example web app

### CODE REVIEW

**Review by abretonc7s — CHANGES_REQUESTED** [review]
only minor change to set appMetadata

File: packages/examples/vuejs/src/components/MetaMaskComponent.vue [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  name: 'MetaMaskComponent',
+  data() {
+    return {
+      sdk: null,
+      accounts: null,
+    };
+  },
+  created() {
+    this.sdk = new MetaMaskSDK({
+      enableDebug: true,
```
[the remark this answers is not in the document]
Comment (andreahaku) [reply packages/examples/vuejs/src/components/MetaMaskComponent.vue:28]: added. thanks

**Review by abretonc7s — APPROVED** [review]
LGTM

**Review by abretonc7s — APPROVED** [review]
LGTM

**Review by abretonc7s — APPROVED** [review]
LGTM

**Pull request merged**